### DISCUSSION TITLE
How to import files into Tailwind config (v3.3 with JITI)

### DISCUSSION THREAD

**Discussion by lominming** [tailwindlabs/tailwindcss#11154] [Help] [answered]
Trying to simply import constants into `tailwind.config.ts` (v3.3 with JITI)
My build tool is vite

In `tailwind.config.ts`
```
import { theme } from '~/constants/themes';
```

In `theme.ts`
```
export const theme = {...}
```

Getting this error:

![image](https://user-images.githubusercontent.com/742933/236047933-761160ce-0992-415e-9ca2-ceb5ceac7056.png)

I also tried the `require(...)` with `module.exports` in my theme file, but it still throws the same error.

**Comment by cannap** [comment]
try to import it via ` import { theme } from './to/the/path'`

**Comment by wongjn** [comment, marked as answer]
Consider checking the file you are trying to import is really at `node_modules/~/constants/themes.ts`. Vite module aliases are not used in Tailwind configuration since it is run separately from Vite.

**Comment by lominming** [comment]
I am so used to it that I forgot that I have Vite alias setup. Relative path works. Thank you!
